index.html

```diff
@@ -31,24 +31,230 @@ <h1>Multi-Post<br> Stories</h1>
+                <li class="js"><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="see-project-btn">
+              <a href="#">See Projects</a>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="story-wrapper">
+        <div class="work-section">
+          <div class="story-snapshoot-potofolio"></div>
+          <div class="work-txt">
+            <h1 class="header-1">Multi-Post <br />Stories</h1>
+            <ul class="list-project-story">
+              <li class="first-element-ul">CANOPY</li>
+              <li><img src="/assets/images/Counter.png" alt="counter" /></li>
+              <li>Back End Dev</li>
+              <li><img src="/assets/images/Counter.png" alt="counter" /></li>
+              <li>2015</li>
+            </ul>
+            <div class="work-paragraph">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="work-buttons">
+              <ul>
+                <li class="html"><a href="#">html</a></li>
+                <li class="css"><a href="#">css</a></li>
+                <li class="js"><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="see-project-btn">
+              <a href="#">See Projects</a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="work-wrapper">
+        <div class="work-section">
+          <div class="snapshoot-potofolio2"></div>
+          <div class="work-txt">
+            <h1>Tonic</h1>
+            <ul class="list-project">
+              <li class="first-element-ul">CANOPY</li>
+              <li><img src="/assets/images/Counter.png" alt="counter" /></li>
+              <li>Back End Dev</li>
+              <li><img src="/assets/images/Counter.png" alt="counter" /></li>
+              <li>2015</li>
+            </ul>
+            <div class="work-paragraph">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="work-buttons">
+              <ul>
+                <li class="html"><a href="#">html</a></li>
+                <li class="css"><a href="#">css</a></li>
+                <li class="js"><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="see-project-btn">
+              <a href="#">See Projects</a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="story-wrapper">
+        <div class="work-section">
+          <div class="story-snapshoot-potofolio2"></div>
+          <div class="work-txt">
+            <h1 class="header-1">Multi-Post <br />Stories</h1>
+            <ul class="list-project-story">
+              <li class="first-element-ul">CANOPY</li>
```

Comment:
- [ ] Following the design, Canopy is also a subtitle, not a list element. Please use a `hx` tag in this case too.

Reply:
write h5 for  subtitle CANPOY